pagopa/pn-frontend
# fix(PN-9412): fix alert message when AAR not available for PA

## Short description
implemented alert when document aar is not immediately available after 120 days

## List of changes proposed in this pull request
- Added TimedMessage component in AAR box in order to alert aar is not immediately available

## How to test
Login as PA, open a notification sent 120 days ago from _today_ and try to download the aar document from aar box. An alert will appear in the same section and not at the top of the timeline.
